gradle.properties

```diff
@@ -2,7 +2,7 @@
 org.gradle.jvmargs=-Xmx1G
 
 # Fabric Properties
-	# check these on https://fabricmc.net/use
+	# check these on https://fabricmc.net/versions.html
```

Comment:
I hope this is ok..

Reply:
Yes the use page is pretty useless for this atm